lib/ruby_lsp/requests/completion_resolve.rb

```diff
@@ -60,7 +60,12 @@ def perform
 
         if first_entry.is_a?(RubyIndexer::Entry::Member)
           detail = first_entry.decorated_parameters
-          label = "#{label}#{first_entry.decorated_parameters}"
+          label = +"#{label}#{first_entry.decorated_parameters}"
+        end
+
+        extra_links = if @item.dig(:data, :guessed_type)
+          label << " | guessed receiver: #{owner_name}"
```

Comment:
We may need a link break here or the guessed receiver is hardly visible due to the UI's width:

<img width="80%" alt="Screenshot 2024-06-21 at 18 23 39" src="https://github.com/Shopify/ruby-lsp/assets/5079556/b9de7360-246b-4dfe-bb1e-7b76f9998ecd">

Reply:
I broke into separate lines, but it's still a bit hard to fit everything. Let me know if you have ideas on how to make it shorter.